atomic_scanners/pipeline-scanner/pipeline-scanner

```diff
@@ -1,10 +1,10 @@
 type: scanner
 scanner_name: pipeline-scanner
 image_name: registry.centos.org/pipeline-images/pipeline-scanner
-default_scan: redhat-release
+default_scan: pipeline-scanner
 scans: [
-  { name: redhat-release,
+  { name: pipeline-scanner,
     args: ['python', 'scanner.py', 'release'],
-    description: "Find Red Hat release"
```

Comment:
Right that will require us to change the scanner itself as well along with the worker. But we can take care of it when we start adding additional scan type to it. In the meanwhile, changing the `name` of scan is not going to involve any additional work.

Umm, I didn't mention anything about the `description`. That looks good to me! :+1: